packages/orchestration/src/chain-capabilities.js

```diff
@@ -57,7 +58,7 @@ const IcaEnabled = /** @type {const} */ ({
 harden(IcaEnabled);
 
 /**
- * @param {Record<string, CosmosChainInfo>} chainInfo
+ * @param {Record<string, ChainInfo>} chainInfo
```

Comment:
Looks like this widening of the parameter is widening the return type. Consider parameterizing the type with generics. It would also help to have a types test for what should be returned.